src/jobs/rspec.yml

```diff
@@ -0,0 +1,45 @@
+description: "Run tests with RSpec"
+
+parameters:
+  executor:
+    description: "Executor to use for this job"
+    type: executor
+  db-port:
+    description: "Waiting for DB to be ready"
```

Comment:
This message looks better.
`"Port to wait for DB startup"`

What do you think?

Reply:
Thanks for your review.
I fixed and pushed.